appinfo/info.xml

```diff
@@ -5,76 +5,64 @@
+- **Copy/Move:** Shift files between any external storage and decentralized storage
 
-Using the plugin, the main file operations that are available in Nextcloud are:
+### **Settings**
 
-- Upload file(s) to a Bee node
-- Download and view files
-- Copy/Move files from any External Storage to a Bee node
-- Copy files from a Bee node to any External Storage
+Users with administrative credentials can configure the plugin through the standard External Storage settings. Here, you can input your license key and URL for Hejbit.
 
-### Settings
+### **Using Hejbit Swarm**
 
-Available to users with administrative credentials, the external storage accepts configuration options from two places, the normal External Storage settings, which allows a user to configure the basic connection properties of a Bee node.
+Once setup and configured, the Decentralized Storage is ready to use.
 
-Since every operation in Swarm costs money, it is important to view how much money is available to the Bee node directly and also provide a convenient way of handling the payment system in Swarm - all managed directly in Nextcloud. This is in a specific section with more advanced configuration for the Bee node, where the following settings can be viewed and edited:
+- Access the Decentralized Storage folder under "All Files" or via the "Files" menu under "External Storage."
+- Once uploaded, it can be viewed internally or downloaded. It can also be copied to another storage.
+- Files from another Nextcloud storage can also be copied/moved to the decentralized storage. Choose the decentralized storage folder as the target Folder.
+- Right-click on a Swarm file to copy the Swarm reference (hash) to your clipboard. Alternatively, click the three dots in the Actions menu and select 'Copy Swarm Reference.' The Swarm reference is the unique address of the file on the Swarm network.
 
-- Configure which Bee node to manage
-- View current status of the Bee node
-- Option to toggle encryption on and off
-- How much BZZ is available to the node
-- Any purchased stamp batches and the remaining balances
-- Have the option to purchase a new batch of stamps
-- Toggle the batch used for uploading files active/inactive
+### **Hejbit Licence**
 
-### Using Swarm
+Experience the future of data storage with 5GB of free, decentralized storage on the Ethereum Swarm network. Our plugin seamlessly integrates with your existing Nextcloud interface, providing true data sovereignty without the complexity.
 
-Once setup and configured, the Bee node is ready to use.
-- Navigate to "Files" option on the menu -> "External Storage" and then the name of the Bee node.
-- You can then upload files just like you normally would in Nextcloud.
-- Once uploaded, it can be viewed internally or downloaded. It can also be copied to another storage.
-- Files from another Nextcloud storage can also be copied/moved to the Bee node.
+[Get your free licence here!](https://metaprovide.org/hejbit/start) .
```

Comment:
@frozenbanana The point after the link was only a fix to prevent opening the thumbnail on collectives. You can remove it from here.